ssas/hash_test.go

```diff
@@ -65,6 +67,35 @@ func (s *HashTestSuite) TestHashWithIter() {
 	s.True(h1.IsHashOf(val))
 }
 
+func TestHashIterTime(t *testing.T) {
+	// This test is not intended to run with the remain suite.  Is is solely used to determine the number of hash
+	// iterations needed to reach a 1 second duration. To invoke the test use:
+	// DEBUG=true  go test -v github.com/CMSgov/bcda-ssas-app/ssas -run TestHashIterTime
+
+	// Check for local dev environment
+	if os.Getenv("DEBUG") != "true" {
+		t.SkipNow()
+	}
+
+	var resultTime int64
```

Comment:
Nit - You can actually keep the resultTime as a time.Duration value (since [time.Duration is also an int64](https://golang.org/pkg/time/#Duration)).

If you keep resultTime as a time.Duration value, you can leverage [time.Duration's String() function](https://golang.org/pkg/time/#Duration.String) which will pretty print the units for you. That way you don't have to assume ms units.

If you decide on that approach, you'll want to cast runCount to a time.Duration value when you are computing the average.

Reply:
Updated resultTime(totalTime) to be defined as a time.Duration type.